docs/01-app/03-api-reference/03-file-conventions/middleware.mdx

```diff
@@ -10,6 +10,8 @@ related:
 
 The `middleware.js|ts` file is used to write [Middleware](/docs/app/getting-started/route-handlers-and-middleware#middleware) and run code on the server before a request is completed. Then, based on the incoming request, you can modify the response by rewriting, redirecting, modifying the request or response headers, or responding directly.
```

Comment:
We should clone this file so the new API name is discoverable instead and not under `middleware` URL.